.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     services:
       postgres:
-        image: postgres:9.5
+        image: postgres:14
```

Comment:
Consistent with [PR already open in Klaxon repo](https://github.com/themarshallproject/klaxon/compare/develop...chore-cleanup-fixes)